elements/pfe-dropdown/src/pfe-dropdown-item.html

```diff
@@ -0,0 +1,3 @@
+<li class="pfe-dropdown-item__container" id="pfe-dropdown-item">
```

Comment:
Using an ID on the list items might not be ideal; since each ID on a page should be unique.

Reply:
Oh yes, that's not going to work. Good catch! 👯